buffer/folder.tf

```diff
@@ -1,18 +1,18 @@
 locals {
-  create_folders = var.enable && var.parent_folder_id != ""
+  create_folders = var.enable && var.root_folder_id != ""
```

Comment:
do you think it is possible that we may need multiple root ids?  i.e., Buffer clients comes want their projects created within different organiation/folders?

Reply:
I guess it is possible, but I think of the root mapping directly to the environment. I say cross that bridge when we come to it.